src/webfaf/utils.py

```diff
@@ -374,3 +376,30 @@ def stream_template(template_name, **context):
     rv = t.stream(context)
     rv.enable_buffering(2)
     return rv
+
+
+def delete_user_bugzillas(db, user_id):
```

Comment:
wondering about this - do we need to remove bugzilla? Could not we only remove the user? I mean, we could only forget who created the bz, who made what changes (history) etc. but not remove the bz from faf and from report?

Reply:
How about removing the user, his attachments and comments (these are not needed? https://github.com/abrt/faf/issues/595) and also the ccs (useless information without the user).
And anonymize the bugzilla and the history.

Does that make sense?

Reply:
yeah, that makes more sense. I just don't want to remove the bugzzilla ID and resolution, as it is rather usefull. And just showing, that bugzilla 123 is related to problem 456 is not showing any personal info.